tests/fuzz/s2n_kyber_r3_recv_public_key_fuzz_test.c

```diff
@@ -25,11 +25,19 @@
  * hex-encoded bytes. This is how we would expect it to appear on the wire. */
 static struct s2n_kem_params kyber512_r3_draft0_params = { .kem = &s2n_kyber_512_r3, .len_prefixed = true };
 static struct s2n_kem_params kyber512_r3_draft5_params = { .kem = &s2n_kyber_512_r3, .len_prefixed = false };
+#if defined(S2N_LIBCRYPTO_SUPPORTS_KYBER)
+static struct s2n_kem_params kyber768_r3_draft5_params = { .kem = &s2n_kyber_768_r3, .len_prefixed = false };
```

Comment:
Every new #if is another action item for https://github.com/aws/s2n-tls/issues/4100 😔 But I think all of the ones in this PR should go away pretty easily.

Reply:
yep, i agree.